### PR TITLE
Update the docstrings to match the actual code defaults

### LINKED ISSUE #8689: Incorrect gamma default values in unified_focal_loss.py docstrings
## Description

The docstrings in `monai/losses/unified_focal_loss.py` have incorrect default values for the `gamma` parameter. The code defaults are correct (matching the [original author's implementation](https://github.com/mlyg/unified-focal-loss/blob/main/loss_functions.py) referenced in #4413), but the docstrings incorrectly state `0.75` for all classes.

## Comparison with Original Implementation

| Loss Function | Original Default γ | MONAI Code | MONAI Docstring |
|---------------|-------------------|------------|-----------------|
| `AsymmetricFocalLoss` | **2.0** | `gamma: float = 2` ✓ | "Defaults to 0.75" ✗ |
| `AsymmetricFocalTverskyLoss` | **0.75** | `gamma: float = 0.75` ✓ | "Defaults to 0.75" ✓ |
| `AsymmetricUnifiedFocalLoss` | **0.5** | `gamma: float = 0.5` ✓ | "Defaults to 0.75" ✗ |

## Specific Issues

1. **AsymmetricFocalLoss** (line 103, 111):
   - Code: `gamma: float = 2`
   - Docstring says: `Defaults to 0.75`

2. **AsymmetricUnifiedFocalLoss** (line 162, 171):
   - Code: `gamma: float = 0.5`
   - Docstring says: `Defaults to 0.75`

3. **AsymmetricUnifiedFocalLoss** also documents an `epsilon` parameter (line 172) that doesn't exist in the `__init__` signature.

## References

- Original paper: [Unified Focal loss: Generalising Dice and cross entropy-based losses](https://arxiv.org/abs/2102.04525) by Michael Yeung et al.
- Original implementation (referenced in #4413): https://github.com/mlyg/unified-focal-loss/blob/main/loss_functions.py

## Suggested Fix

Update the docstrings to match the actual code defaults (which are correct per the original implementation).

I am happy to create a PR if this fix is acceptable.

### PR DESCRIPTION
**Pull request by ytl0623**
Fixes #8689

### Description

Update the docstrings to match the actual code defaults.

1. AsymmetricFocalLoss (line 103, 111): `gamma`
2. AsymmetricUnifiedFocalLoss (line 162, 171): `gamma`
3. AsymmetricUnifiedFocalLoss (line 172): `epsilon`

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests  --disttests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: monai/losses/unified_focal_loss.py [hunk 1]
```diff
@@ -108,7 +108,7 @@ def __init__(
         Args:
             to_onehot_y : whether to convert `y` into the one-hot format. Defaults to False.
             delta : weight of the background. Defaults to 0.7.
-            gamma : value of the exponent gamma in the definition of the Focal loss  . Defaults to 0.75.
+            gamma : value of the exponent gamma in the definition of the Focal loss  . Defaults to 2.
```
Comment (ericspod) [inline monai/losses/unified_focal_loss.py]:
```suggestion
            gamma : value of the exponent gamma in the definition of the Focal loss. Defaults to 2.
```

File: monai/losses/unified_focal_loss.py [hunk 2]
```diff
@@ -168,9 +168,8 @@ def __init__(
             to_onehot_y : whether to convert `y` into the one-hot format. Defaults to False.
             num_classes : number of classes, it only supports 2 now. Defaults to 2.
             delta : weight of the background. Defaults to 0.7.
-            gamma : value of the exponent gamma in the definition of the Focal loss. Defaults to 0.75.
-            epsilon : it defines a very small number each time. simmily smooth value. Defaults to 1e-7.
-            weight : weight for each loss function, if it's none it's 0.5. Defaults to None.
+            gamma : value of the exponent gamma in the definition of the Focal loss. Defaults to 0.5.
```
Comment (ericspod) [inline monai/losses/unified_focal_loss.py]: Please check that the argument descriptions are in the same order as declared.

**Review by ericspod — APPROVED** [review]
Hi @ytl0623 I had minor comments but otherwise it looks good to me.

**Comment by ytl0623** [comment]
Hi @ericspod, please take a look at the code when you're free. Once the CI/CD is green, we can go ahead and merge it. (Similar fixes in other PRs are being merged soon as well.)

Thanks in advance!

**Comment by ericspod** [comment]
Hi @ytl0623 I will soon have a chance to come back to this, sorry for the delay but we have another CI issue with our Docker image we're sorting.

File: monai/losses/unified_focal_loss.py (hunk 2 above)
Comment (ericspod) [reply monai/losses/unified_focal_loss.py]: Sorry, I meant to say here that the arguments in the docstring should match the order as the arguments are declared. Please change the declared arguments back to what they were (so that anyone instantiating this class with positional arguments won't have an issue) and then reorder the descriptions in the docstring to match.

File: monai/losses/unified_focal_loss.py [hunk 3]
```diff
@@ -44,8 +44,8 @@ def __init__(
         Args:
             to_onehot_y: whether to convert `y` into the one-hot format. Defaults to False.
             delta : weight of the background. Defaults to 0.7.
-            gamma : value of the exponent gamma in the definition of the Focal loss  . Defaults to 0.75.
-            epsilon : it defines a very small number each time. similarly smooth value. Defaults to 1e-7.
+            gamma : value of the exponent gamma in the definition of the Focal loss. Defaults to 0.75.
+            epsilon : it defines a very small number each time. simmily smooth value. Defaults to 1e-7.
```
Comment (ericspod) [inline monai/losses/unified_focal_loss.py]: Sorry, I might have just introduced the misspelling here trying to merge from dev, it should be "similarly" not "simmily" here and in the next change on line 112.

File: monai/losses/unified_focal_loss.py (hunk 2 above)
Comment (ytl0623) [reply monai/losses/unified_focal_loss.py]: Done, thank u!

File: monai/losses/unified_focal_loss.py (hunk 3 above)
Comment (ytl0623) [reply monai/losses/unified_focal_loss.py]: Done.
